File: CS/dxSample/Data/Category.cs
using System.Collections.Generic;
using System.ComponentModel;
using System;

namespace dxSample.Data {
    public class Category {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
    }
    public static class CategoryData {
        static Random _random = new Random();
        static string[] _possibleItems = new string[] { "Tea", "Coffee", "Butter", "Lemon", "Apple", "Orange", "Milk", "Chocolate" };
        static string[] _categories = new string[] { "Main", "New", "Best" };
        public static IList<Category> GetCategories() {
            BindingList<Category> categories = new BindingList<Category>();
            for (int i = 0; i < 8; i++) {
                int index = i + 1;
                string items = string.Empty;
                int itemsCount = _random.Next(4, 14);
                for (int j = 0; j < itemsCount; j++) {
                    items += _possibleItems[_random.Next(4, 8)];
                    items += (j < itemsCount - 1 ? ", " : "");
                }
                categories.Add(new Category() { Id = i, Name = _categories[_random.Next(0, 3)], Description = items });
            }
            return categories;
        }
    }
}

File: VB/dxSample/Data/Category.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System

Namespace dxSample.Data

    Public Class Category

        Public Property Id As Integer

        Public Property Name As String

        Public Property Description As String
    End Class

    Public Module CategoryData

        Private _random As Random = New Random()

        Private _possibleItems As String() = New String() {"Tea", "Coffee", "Butter", "Lemon", "Apple", "Orange", "Milk", "Chocolate"}

        Private _categories As String() = New String() {"Main", "New", "Best"}

        Public Function GetCategories() As IList(Of Category)
            Dim categories As BindingList(Of Category) = New BindingList(Of Category)()
            For i As Integer = 0 To 8 - 1
                Dim index As Integer = i + 1
                Dim items As String = String.Empty
                Dim itemsCount As Integer = _random.Next(4, 14)
                For j As Integer = 0 To itemsCount - 1
                    items += _possibleItems(_random.Next(4, 8))
                    items += If(j < itemsCount - 1, ", ", "")
                Next

                categories.Add(New Category() With {.Id = i, .Name = _categories(_random.Next(0, 3)), .Description = items})
            Next

            Return categories
        End Function
    End Module
End Namespace
